upstream/CVE-2017-1002101.json

```diff
@@ -0,0 +1,89 @@
+{
+  "id": "CVE-2017-1002101",
+  "modified": "2018-03-05T20:53:58Z",
+  "published": "2018-03-05T20:53:58Z",
+  "summary": "subpath volume mount handling allows arbitrary file access in host filesystem",
+  "details": "In Kubernetes versions 1.3.x, 1.4.x, 1.5.x, 1.6.x and prior to versions 1.7.14, 1.8.9 and 1.9.4 containers using subpath volume mounts with any volume type (including non-privileged pods, subject to file permissions) can access files/directories outside of the volume, including the host's filesystem.",
+  "affected": [
+    {
+      "package": {
+        "ecosystem": "kubernetes",
+        "name": "k8s.io/kubernetes"
```

Comment:
after looking at merged PRs its look like fix has been made in to packages :
- apiserver
- kubelet

Reply:
fixed